### PR TITLE
Converts many proc overrides to properly use list/modifiers, lots of other smaller things

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

Converts many proc overrides to properly use list/modifiers, fixes some spots where modifiers should have been passed, calls modifiers what it is, a lazy list, and cleans up some improper arg names like L, M, C, and N. Oh and I think there was a spot where someone was trying to pass M.name in as a string, but forgot to wrap it in []. I fixed that too.

## Why It's Good For The Game

I am in great pain. Also fuck you, have a free merge conflict

### CODE REVIEW

File: code/game/objects/structures/spawner.dm [hunk 1]
```diff
@@ -19,10 +19,10 @@
 	. = ..()
 	AddComponent(spawner_type, mob_types, spawn_time, faction, spawn_text, max_mobs)
 
-/obj/structure/spawner/attack_animal(mob/living/simple_animal/M)
-	if(faction_check(faction, M.faction, FALSE)&&!M.client)
+/obj/structure/spawner/attack_animal(mob/living/simple_animal/user, list/modifiers)
+	if(faction_check(faction, user.faction, FALSE) && !user.client)
 		return
-	..()
+	..() //wonder if this is intended
```
Comment (Rohesie) [inline code/game/objects/structures/spawner.dm:25]:
```suggestion
	return ..() //wonder if this is intended
```

File: code/game/turfs/closed/minerals.dm [hunk 2]
```diff
@@ -101,17 +101,17 @@
...
 
-/turf/closed/mineral/attack_alien(mob/living/carbon/alien/M)
-	to_chat(M, "<span class='notice'>You start digging into the rock...</span>")
+/turf/closed/mineral/attack_alien(mob/living/carbon/alien/user, list/modifiers)
+	to_chat(user, "<span class='notice'>You start digging into the rock...</span>")
 	playsound(src, 'sound/effects/break_stone.ogg', 50, TRUE)
-	if(do_after(M, 40, target = src))
-		to_chat(M, "<span class='notice'>You tunnel into the rock.</span>")
-		gets_drilled(M)
+	if(do_after(user, 40, target = src))
```
Comment (Rohesie) [inline code/game/turfs/closed/minerals.dm:112]:
```suggestion
	if(do_after(user, 4 SECONDS, target = src))
```

File: code/modules/clothing/masks/miscellaneous.dm [hunk 3]
```diff
@@ -8,10 +8,10 @@
 	gas_transfer_coefficient = 0.9
 	equip_delay_other = 20
 
-/obj/item/clothing/mask/muzzle/attack_paw(mob/user)
+/obj/item/clothing/mask/muzzle/attack_paw(mob/user, list/modifiers)
 	if(iscarbon(user))
-		var/mob/living/carbon/C = user
-		if(src == C.wear_mask)
+		var/mob/living/carbon/lad = user
+		if(src == lad.wear_mask)
```
Comment (Rohesie) [inline code/modules/clothing/masks/miscellaneous.dm:14]:
```suggestion
		var/mob/living/carbon/carbon_user = user
		if(src == carbon_user.wear_mask)
```

Comment (LemonInTheDark) [reply code/modules/clothing/masks/miscellaneous.dm:14]: I like mine better but ok

**Comment by Rohesie** [comment]
Set it back to ready for review after testing it locally.
Code looks fine, though, don't expect anything to break.

**Comment by LemonInTheDark** [comment]
Tested locally, obviously can't test each case, but nothing springs out at me as broken

**Comment by Rohesie** [comment]
Lots of conflicts. Do give it a quick test after resolving them.

**Comment by LemonInTheDark** [comment]
Works from what I can see

**Comment by LemonInTheDark** [comment]
Oh come on

**Comment by LemonInTheDark** [comment]
Still works

**Comment by Cyberboss** [comment]
Yeehaw

**Pull request merged**